frontend/src/components/common/PieChart.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <GChart
+        type="PieChart"
+        :data="chartData"
+        :options="chartOptions"
+    />
+</template>
+
+<script>
+    import {GChart} from 'vue-google-charts'
+
+    export default {
+        components: {
+            GChart
+        },
+        props: {
+            title: {
+                type: String,
+                default: 'PieChart'
+            },
+            data: {
+                type: Array,
+                default: () => []
```

Comment:
make this prop required

Reply:
Fixed

Reply:
when prop is required, `default` is redundant

Reply:
Fixed